services/core-web/src/components/mine/DigitalPermitCredential/ViewDigitalPermitCredential.tsx

```diff
@@ -0,0 +1,344 @@
+
+  const handleRevoke = async (data) => {
+    if (connectionDetails.length < 1) return;
+
+    await dispatch(
+      revokeCredential({
+        partyGuid: digitalPermitCredential.current_permittee_guid,
+        comment: data.comment,
+        credential_exchange_id: connectionDetails[0].cred_exch_id,
+      })
+    );
+
+    dispatch(closeModal());
+  };
+
+  const openRevokeDigitalCredentialModal = (event) => {
+    event.preventDefault();
+    dispatch(
+      openModal({
+        props: {
+          initialValues: {},
+          onSubmit: handleRevoke,
+          title: "Revoke Digital Permit Credential",
+          mineGuid,
+        },
+        width: "50vw",
+        content: modalConfig.REVOKE_CREDENTIAL_MODAL,
+      })
+    );
+  };
+
+  return (
+    <div className="tab__content margin-large--top">
+      {VC_CRED_ISSUE_STATES[connectionDetails[0]?.cred_exch_state] ===
+        VC_CRED_ISSUE_STATES.credential_revoked && (
+        <Alert
+          className="margin-large--bottom"
+          description={
+            <Row justify="space-between" align="middle">
+              <Paragraph strong>This digital credential was revoked</Paragraph>
+              <Button type="default" className="no-bg">
+                Re-offer Credential
+              </Button>
+            </Row>
+          }
+          type="error"
+          showIcon
+        />
+      )}
+      <Title level={2}>Digital Permit Credential</Title>
+      <Row gutter={48}>
+        <Col md={12} sm={24}>
+          <Title level={3} className="primary-colour">
+            Credential Details
+          </Title>
+          <>
+            <Row gutter={6}>
+              <Col span={12}>
+                <Paragraph strong>Permittee Name</Paragraph>
+                <Paragraph>{digitalPermitCredential?.current_permittee}</Paragraph>
+              </Col>
+              <Col span={12}>
+                <Paragraph strong>Issue Date</Paragraph>
+                <Paragraph>{formatDate(getLatestIssueDate())}</Paragraph>
+              </Col>
+            </Row>
+          </>
+
+          <Row gutter={6}>
+            <Col span={12}>
+              <Paragraph strong>Permit Number</Paragraph>
+              <Paragraph>{digitalPermitCredential?.permit_no}</Paragraph>
+            </Col>
+
+            <Col span={12}>
+              <Paragraph strong>Mine Number</Paragraph>
+              <Paragraph>{mine.mine_no}</Paragraph>
+            </Col>
+          </Row>
+          <Row gutter={6} className="margin-large--bottom">
+            <Col span={12}>
+              <Paragraph strong>Latitude</Paragraph>
+              <Paragraph>{mine.mine_location.latitude}</Paragraph>
+            </Col>
+            <Col span={12}>
+              <Paragraph strong>Longitude</Paragraph>
+              <Paragraph>{mine.mine_location.longitude}</Paragraph>
+            </Col>
+          </Row>
+          <ExplosivesPermitMap pin={[mine.mine_location.latitude, mine.mine_location.longitude]} />
```

Comment:
wait- I remember mine_location missing from the API call on the MS My Mines page- does this mean it's fixed or was that from a different endpoint?

Reply:
It's there.  I'm not sure if it specifically got fixed or not.  Maybe it's a Minespace vs. Core thing?